jina/jaml/parsers/executor/legacy.py

```diff
@@ -1,57 +1,14 @@
 import dataclasses
-import inspect
-from functools import reduce
-from typing import Any, Dict, Optional, Set, Type
+from typing import Any, Dict, Optional, Type
 
-from jina.jaml.parsers.base import VersionedYAMLParser
+from jina.jaml.parsers.base import BaseLegacyParser
 from jina.serve.executors import BaseExecutor
 from jina.serve.executors.metas import get_default_metas
 
 
-class LegacyParser(VersionedYAMLParser):
+class ExecutorLegacyParser(BaseLegacyParser):
```

Comment:
Executor? but also works for `Flow` right?

Reply:
no Flow parser is called V1Parser with different behavior